cmd/fdb/backup_list.go

```diff
@@ -16,7 +17,7 @@ var backupListCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		folder, err := internal.ConfigureFolder()
 		tracelog.ErrorLogger.FatalOnError(err)
-		internal.DefaultHandleBackupList(folder)
+		internal.DefaultHandleBackupList(folder, utility.BaseBackupPath)
```

Comment:
Maybe I missed that point, but why we can't instead call this function like this?
```
internal.DefaultHandleBackupList(folder.GetSubFolder(utility.BaseBackupPath))
```

Reply:
Ah fair point. I'll update